MatrixSDKExtensions/VoIP/Jingle/MXJingleCallStackCall.m

```diff
@@ -159,22 +164,48 @@ - (void)handleRemoteCandidate:(NSDictionary<NSString *, NSObject *> *)candidate
     RTCIceCandidate *iceCandidate = [[RTCIceCandidate alloc] initWithSdp:(NSString *)candidate[@"candidate"]
                                                            sdpMLineIndex:[(NSNumber *)candidate[@"sdpMLineIndex"] intValue]
                                                                   sdpMid:(NSString *)candidate[@"sdpMid"]];
-    [peerConnection addIceCandidate:iceCandidate];
+
+    // Ice candidates have to be added after the remote description has been set
+    if (!peerConnection.remoteDescription)
+    {
+        // Cache ice candidates until remote description is set
+        if (iceCandidateCache == nil)
+        {
+            iceCandidateCache = [NSMutableArray arrayWithObject:iceCandidate];
+        }
+        else
+        {
+            [iceCandidateCache addObject:iceCandidate];
+        }
```

Comment:
@Legi429, here the first iceCandidate is being ignored, because of the else condition, if the cache is nil.

Reply:
I think this should still work since I'm initializing the array _with_ the first iceCandidate for that reason. Or am I missing something maybe?

Reply:
Sorry @Legi429, you're right. It seems I need a break.